nekoyume/Assets/_Scripts/UI/Tween/DigitTextTweener.cs

```diff
@@ -1,65 +1,60 @@
-using System;
 using DG.Tweening;
 using TMPro;
 using UnityEngine;
 
 namespace Nekoyume.UI.Tween
 {
     [RequireComponent(typeof(TextMeshProUGUI))]
-    public class DigitTextTweener : MonoBehaviour
+    public class DigitTextTweener : DOTweenBase
     {
         public TweenCallback onComplete = null;
 
-        public int startValue = 0;
+        public int beginValue = 0;
 
         public int endValue = 0;
 
-        [SerializeField]
-        protected float duration = 0.0f;
+        private TextMeshProUGUI _text = null;
 
-        private TextMeshProUGUI text = null;
-
-        protected Tweener Tweener { get; set; }
-
-        private void Awake()
+        protected override void Awake()
         {
-            text = GetComponent<TextMeshProUGUI>();
+            base.Awake();
+            _text = GetComponent<TextMeshProUGUI>();
         }
 
         private void OnDisable()
         {
             KillTween();
         }
 
-        public Tweener Play()
+        public new DG.Tweening.Tween Play()
```

Comment:
Why did you overlap this with `new`, not using `override`?

Reply:
Previously, it was a code that returned Tweener.
This part was created by changing base class of this class from `BaseTweener` to `DOTweenBase`.
But now I don't need it, so I'm fixing it.